pkgs/top-level/release-haskell.nix

```diff
@@ -93,6 +96,16 @@ let
         writers = testPlatforms.writers;
       };
 
+      # test some statically linked packages to catch regressions
+      # and get some cache going for static compilation with GHC
+      pkgsStatic.haskellPackages = {
+        inherit (staticHaskellPackagesPlatforms)
```

Comment:
any suggestions on changes to this list? It shouldn't be too big, but some packages which are a) important and b) unlikely to break (or will degrade the quality of haskellPackages significantly if they break). Also maybe some kind of smaller tool which is a bit more complicated than `hello` could be a good idea.

Reply:
not sure if that fits, but `pandoc` came to my mind

Reply:
but probably not exactly a "small" tool..

Reply:
I think this list looks good for now.

In https://github.com/NixOS/nixpkgs/pull/122586#issuecomment-846400891, @rnhmjoj mentioned an issue about packages using TemplateHaskell.  I wonder if `lens` uses TemplateHaskell anywhere.

Well, I'll go ahead and merge this in, and we can see what happens.